lib/newrelic-couchbase/instrumentation/couchbase.rb

```diff
@@ -1,18 +1,15 @@
 ::Couchbase.module_eval do
   class << self
-    include NewRelic::Agent::MethodTracer
-
     [
+      :new,
```

Comment:
What was the reason for adding `new` here?

Reply:
I think the `new` call was being recorded as `connect` below. Just moved it up here.